Jenkinsfile

```diff
@@ -133,6 +133,8 @@ if (env.CHANGE_TITLE && env.CHANGE_TITLE.contains('[ci-skip]')) {
                                             sh "npm run vendor-install"
                                             sh "unzip ${env.KEYBASE_JS_VENDOR_DIR}/flow/flow-linux64*.zip -d ${env.BASEDIR}"
                                             sh "${env.BASEDIR}/flow/flow status shared"
+                                            sh "npm run shrinkwrap-audit"
```

Comment:
can this happen in parallel with flow? Is it dependent on vendor-install? If it isn't dependent, can do:

```groovy
parallel (
    test_linux_js_flow: {
        dir("desktop") {
            <flow stuff here...>
        }
    },
    test_linux_js_shrinkwrap: {
        sh "npm run shrinkwrap-audit -- ./desktop"
        sh "npm run shrinkwrap-audit -- ./react-native"
    },
)
```